src/ls_server.rs

```diff
@@ -61,8 +62,8 @@ enum Method {
 
 #[derive(Debug)]
 enum Notification {
-    CancelRequest(usize),
-    Change(ChangeParams),
+    CancelRequest(NumberOrString),
```

Comment:
Is this an id? I think there is another PR that uses `Id` rather than NumberOrString, it would be good to rebase on top of that